### PR TITLE
[RHMIDOC-213] - Release Notes RHMI 2.6.0

### PR DESCRIPTION
**Pull request by joaedwar**
# Description
The release notes (KSC article) for the RHMI 2.6.0 release.

* https://issues.redhat.com/browse/RHMIDOC-213
* https://issues.redhat.com/browse/RHMIDOC-214 
* https://issues.redhat.com/browse/RHMIDOC-220
* https://issues.redhat.com/browse/RHMIDOC-227

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist
- [ ] This change requires a documentation update 
- [x] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added a test case that will be used to verify my changes 
- [ ] Verified independently on a cluster by reviewer

### CODE REVIEW

**Comment by sjay72** [comment]
Note for posterity - It seems like this docs test case PR will not be relevant for a 2.6.0 release, but perhaps for a 2.6.1 update instead. We don't yet know a code freeze date for 2.6.1.

File: test-cases/tests/documentation/i01-verify-release-notes.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+
+## Description
+
+The release notes (KSC article) for the RHMI 2.6.0 release.
+
+- Jira ref link: https://issues.redhat.com/browse/RHMIDOC-213, https://issues.redhat.com/browse/RHMIDOC-214, https://issues.redhat.com/browse/RHMIDOC-220, https://issues.redhat.com/browse/RHMIDOC-227
+
+- Peer contact: Susan Jay; Joan Edwards
+
+* Staging doc link: https://access.redhat.com/articles/5224991/draft
```
Comment (trepel) [inline test-cases/tests/documentation/i01-verify-release-notes.md:18]: This link no longer works. Is there any other that could be used?

**Pull request closed without merging**